src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -1978,5 +1978,63 @@ public static void setShowStandingsPanes(boolean showStandingsPanes) {
     public static boolean isShowStandingsPanes() {
         return showStandingsPanes;
     }
+    
+    /**
+     * Concatenate Arrays.
+     * 
+     * This is null-safe, arrays can be null and will return 
+     * an array.
+     * 
+     * @param one first array
+     * @param two secodn array
+     * @return a new array which contains contents of one and two arrays
+     */
+    public static String[] copyArray(String[] one, String[] two) {
```

Comment:
As the header comment states, the function of this method is to CONCATENATE two arrays.  However, the name of the method (`copyArray`) implies that a _single_ array is being copied.  If I just look at the method signature, including the name and the fact that it has two parameters, I would assume it copies the first array into the second one (or maybe vice-versa).  I think the method name should reflect what it actually does; it should be named (something like) `concatenateArrays()`.

Reply:
pushed fix.